employees/templates/employees/partial/display_reports/reports_in_date_for_author.html

```diff
@@ -18,7 +19,15 @@
     </td>
     <td class="edit-button-column Invisible hidden-print">
         {% if report.editable %}
-            <a href="{% if request.user.is_admin %}{% url 'admin-report-detail' pk=report.id %}{% else %}{% url 'custom-report-detail' pk=report.id %}{% endif %}" class="btn btn-light hidden-print">
+            {% if request.user.is_admin%}
+                {% if request.path == custom_report_list_url %}
```

Comment:
Here should not be checked if this url match. Here should be checked if report which user want to edit is his. Please change this condition to check if this report is user's.

Reply:
done